master/networking/mtu.md

```diff
@@ -3,127 +3,138 @@ title: Configuring MTU
 
-### Setting tunnel MTU with calicoctl
+- If you use VXLAN anywhere in your pod network, configure MTU size as “physical network MTU size minus 50”. 
+- If you use only IP in IP, configure MTU size as “physical network MTU size minus 20”
+- Set the workload endpoint MTU and the tunnel MTUs to the same value (so all paths have the same MTU)
 
-To set the IP-in-IP MTU value for all {{site.prodname}} nodes in your cluster, use the
-following command to retrieve the current Felix settings.
+##### MTU for flannel networking
 
-```bash
-calicoctl get felixconfig default --export -o yaml > felix.yaml
-```
+When using flannel for networking, the MTU for network interfaces should match the MTU of the flannel interface. If using flannel with VXLAN, use the “Calico MTU with VXLAN” column in the table above for common sizes. 
+
+#### Configure MTU for workloads
+  
+When you set the MTU it applies to new workloads. To apply MTU changes to existing workloads, you must restart calico nodes. Restarting the calico/node pods also updates any Calico tunnel network interfaces on that node. 
 
-Modify ipipMTU (or vxlanMTU) to the intended integer value.
+Edit `calico-config ConfigMap` with the veth MTU value for your environment. For example: 
 
-```bash
-vim felix.yaml
 ```
+# The CNI network configuration to install on each node.
+  cni_network_config: |-
+    {
+        "name": "k8s-pod-network",
+        "cniVersion": "0.1.0",
+        "type": "calico",
+        "etcd_endpoints": "__ETCD_ENDPOINTS__",
+        "etcd_key_file": "__ETCD_KEY_FILE__",
+        "etcd_cert_file": "__ETCD_CERT_FILE__",
+        "etcd_ca_cert_file": "__ETCD_CA_CERT_FILE__",
+        "log_level": "info",
+        "mtu": 1440,
+```
+
+#### Configure MTU for overlay networking
+
+If you are using IP in IP and/or VXLAN for Calico overlay networking, set the tunnel MTU to match the value that you configured for the veth MTU. 
 
-Replace the current felixconfig settings.
+Edit `calico-config ConfigMap` with the MTU tunnel values for your environment. For example: 
 
-```bash
-calicoctl replace -f felix.yaml
 ```
+# Set MTU for tunnel device used if ipip is enabled
+            - name: FELIX_IPINIPMTU
+              value: "1440"
+# Set MTU for tunnel device used if vxlan is enabled
+           - name: FELIX_VXLAN
+              value: “1440”
+```
+
+##### View existing tunnel MTU values
+
+To view the existing tunnel size, use the command: 
+
+`ip addr sh`
+
+The IP in IP tunnel appears as tunlx (for example, tunl0), along with the MTU size. For example:
+
+![Tunnel MTU]({{site.baseurl}}/images/tunnel.png)
+
+##### Set tunnel MTU values in FelixConfiguration
+
+You can set tunnel MTU using [Felix configuration file](https://docs.projectcalico.org/master/reference/resources/felixconfig):
```

Comment:
Can we drop the word `file`?

Reply:
Agree - we're not talking about the file here, we're talking about the FelixConfiguration API.

Reply:
639383223a74c00248b8d5c2cf5815c67437fc62

Reply:
Fixed.